test/system/010-images.bats

```diff
@@ -241,4 +241,94 @@ Labels.created_at | 20[0-9-]\\\+T[0-9:]\\\+Z
 
     run_podman rmi test:1.0
 }
+
+@test "podman images - remove an image that's lost its manifest" {
+	# the image's ID is known for the image with the digest we're about to pull
+	image=quay.io/libpod/alpine@sha256:634a8f35b5f16dcf4aaa0822adc0b1964bb786fca12f6831de8ddc45e5986a00
+	id=961769676411f082461f9ef46626dd7a2d1e2b2a38e6a44364bcbecf51e66dd4
+	run_podman pull $image
+	run_podman info -f '{{.Store.GraphRoot}}'
+	graphroot="$output"
+	rm -v ${graphroot}/*-images/$id/manifest
+	run_podman 125 images
+	run_podman rmi -f $id
+	if [[ "$PODBIN_NAME" != remote ]] ; then
+		run_podman system reset -f
+	fi
+}
+
+@test "podman images - remove all images when one's lost its manifest" {
+	# the image's ID is known for the image with the digest we're about to pull
+	image=quay.io/libpod/alpine@sha256:634a8f35b5f16dcf4aaa0822adc0b1964bb786fca12f6831de8ddc45e5986a00
+	id=961769676411f082461f9ef46626dd7a2d1e2b2a38e6a44364bcbecf51e66dd4
+	run_podman pull $image
+	run_podman info -f '{{.Store.GraphRoot}}'
+	graphroot="$output"
+	rm -v ${graphroot}/*-images/$id/manifest
+	run_podman 125 images
+	run_podman rmi -f -a
+	if [[ "$PODBIN_NAME" != remote ]] ; then
+		run_podman system reset -f
+	fi
+}
+
+@test "podman images - prune images when one's lost its manifest" {
+	# the image's ID is known for the image with the digest we're about to pull
+	image=quay.io/libpod/alpine@sha256:634a8f35b5f16dcf4aaa0822adc0b1964bb786fca12f6831de8ddc45e5986a00
+	id=961769676411f082461f9ef46626dd7a2d1e2b2a38e6a44364bcbecf51e66dd4
+	run_podman pull $image
+	run_podman info -f '{{.Store.GraphRoot}}'
+	graphroot="$output"
+	rm -v ${graphroot}/*-images/$id/manifest
+	run_podman 125 images
+	run_podman image prune -a -f
+	if [[ "$PODBIN_NAME" != remote ]] ; then
+		run_podman system reset -f
+	fi
+}
+
+@test "podman images - remove an image that's lost its config blob" {
+	# the image's ID is known for the image with the digest we're about to pull
+	image=quay.io/libpod/alpine@sha256:634a8f35b5f16dcf4aaa0822adc0b1964bb786fca12f6831de8ddc45e5986a00
+	id=961769676411f082461f9ef46626dd7a2d1e2b2a38e6a44364bcbecf51e66dd4
+	run_podman pull $image
+	run_podman info -f '{{.Store.GraphRoot}}'
+	graphroot="$output"
+	rm -v ${graphroot}/*-images/$id/=$(echo -n "sha256:$id" | base64 -w0)
+	run_podman 125 images
+	run_podman rmi -f $id
+	if [[ "$PODBIN_NAME" != remote ]] ; then
```

Comment:
Why do you need to reset storage?  And if you don't reset it on remote, why isn't it broken then?

Reply:
Hmm, yeah, that should probably just be a different test.